cl/corpus_importer/management/commands/harvard_opinions.py

```diff
@@ -1000,27 +1000,27 @@ def winnow_case_name(case_name: str) -> Set:
     )
 
     # Fix case name to be cleaner
-    case_name = harmonize(case_name)
+    prepared_case_name = harmonize(case_name)
 
     # Join abbreviations/acronyms. e.g. "D.L.M. v. T.J.S." -> "DLM v. TJS"
-    case_name = re.sub(
+    prepared_case_name = re.sub(
         r"\b[a-zA-Z][a-zA-Z\.]*[A-Za-z]\b\.?",
         lambda m: m.group().replace(".", ""),
-        case_name,
+        prepared_case_name,
     )
 
     # Remove all non-alphanumeric characters
-    case_title = re.sub(r"[^a-z0-9 ]", " ", case_name.lower())
+    case_title = re.sub(r"[^a-z0-9 ]", " ", prepared_case_name.lower())
 
     # Remove strings that can cause an unnecessary overlap
     case_title = false_positive_strings_regex.sub("", case_title)
 
     # Remove one-letter words, initials etc.
     case_title = re.sub(r"\b[^ ]\b", "", case_title)
 
-    if not case_title:
+    if not case_title and case_name:
         # Log case name if the process reduce it to blank
-        logger.warning(f"Case name: {case_name} reduced to blank.")
+        logger.warning(f'Case name: "{case_name}" reduced to blank.')
```

Comment:
I'm not really sure I understand what is going on here.  Why would be also include case_name and instead of prepared_case_name and why would we do both?

Reply:
We log the original case name, not the one reduced to blank, the condition ensures that the name entered is not empty and it was reduced to blank. To avoid additional warnings in the logs, with this, the message will be logged unless is really necessary. I ran the function against all case names from case.law metadata and none throwed a warning. So we could also remove the message.